bridgetown-website/frontend/javascript/index.js

```diff
@@ -52,19 +52,21 @@ document.addEventListener('DOMContentLoaded', () => {
     containers = [mainEl, "#topnav"]
   }
 
-  const swup = new Swup({
-    containers: containers,
-    plugins: [
-      new SwupSlideTheme({mainElement: mainEl}),
-      new SwupBodyClassPlugin(),
-      new SwupScrollPlugin({animateScroll: false})
-    ],
-    
-  })
+  if(!window.matchMedia('(prefers-reduced-motion)')) {
```

Comment:
I think you need a .matches on the end there to account for cases where it's false — right now this seems to be true all the time.

Reply:
@jaredcwhite appears to be correct. As of now it's only testing that the property exists, but doesn't actually test whether or not the property value is true / false.

Just figured I'd also drop the relevant document on `matchMedia` just for future reference in case we ever revisit this.

[https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Testing_media_queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Testing_media_queries)

Thanks again for the PR :)

Reply:
Ah my mistake (classic lunchtime coding!). All sorted :)

Thank you for your help (and awesome responsiveness!) on this PR :)